src/app/app.component.scss

```diff
@@ -14,6 +15,7 @@
   box-sizing: border-box;
   height: 100px;
   background: rgb(39, 39, 42);
+  box-shadow: 0px 0px 8px rgba(0, 0, 0, .6);
```

Comment:
Usually, you don't need to add ps with 0 values

Reply:
done